package.json

```diff
@@ -0,0 +1,176 @@
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/",
+    "test": "node node_modules/.bin/jest --no-cache",
+    "test:watch": "node node_modules/.bin/jest --watchAll",
+    "test:current": "node node_modules/.bin/jest --watch",
+    "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook",
+    "deploy-storybook": "storybook-to-ghpages",
+    "analyze": "./script/webpack-analyze",
+    "create-react-component": "yo react-domain"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/theforeman/foreman-tasks.git"
+  },
+  "bugs": {
+    "url": "http://projects.theforeman.org/projects/foreman-tasks/issues"
+  },
+  "devDependencies": {
+    "@storybook/addon-actions": "~3.4.11",
+    "@storybook/addon-knobs": "~3.4.11",
+    "@storybook/react": "~3.4.11",
+    "@storybook/storybook-deployer": "^2.0.0",
+    "argv-parse": "^1.0.1",
+    "axios-mock-adapter": "^1.10.0",
+    "babel-cli": "^6.10.1",
+    "babel-core": "^6.26.3",
+    "babel-eslint": "^8.2.3",
+    "babel-jest": "^23.6.0",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-dynamic-import-node": "^2.0.0",
+    "babel-plugin-lodash": "^3.3.4",
+    "babel-plugin-module-resolver": "^3.2.0",
+    "babel-plugin-syntax-dynamic-import": "^6.18.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-object-assign": "^6.8.0",
+    "babel-plugin-transform-object-rest-spread": "^6.8.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-register": "^6.9.0",
+    "compression-webpack-plugin": "~1.1.11",
+    "coveralls": "^3.0.0",
+    "css-loader": "^0.23.1",
+    "dotenv": "^5.0.0",
+    "enzyme": "^3.4.0",
+    "enzyme-adapter-react-16": "^1.4.0",
+    "enzyme-to-json": "^3.2.1",
+    "eslint": "^4.10.0",
+    "eslint-plugin-patternfly-react": "0.2.0",
+    "expose-loader": "~0.6.0",
+    "extract-text-webpack-plugin": "^3.0.0",
+    "file-loader": "^0.9.0",
+    "highlight.js": "~9.14.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest-cli": "^23.6.0",
+    "jest-prop-type-error": "^1.1.0",
+    "lodash-webpack-plugin": "^0.11.4",
+    "node-sass": "^4.5.0",
+    "prettier": "^1.13.5",
+    "raf": "^3.4.0",
+    "raw-loader": "^0.5.1",
+    "react-redux-test-utils": "^0.1.1",
+    "react-remarkable": "^1.1.3",
+    "react-test-renderer": "^16.8.1",
+    "redux-mock-store": "^1.2.2",
+    "sass-loader": "~6.0.6",
+    "style-loader": "^0.13.1",
+    "stylelint": "^9.3.0",
+    "stylelint-config-standard": "^18.0.0",
+    "uglifyjs-webpack-plugin": "^1.2.2",
+    "url-loader": "^1.0.1",
+    "webpack": "^3.4.1",
+    "webpack-bundle-analyzer": "^2.13.1",
+    "webpack-dev-server": "^2.5.1",
+    "webpack-stats-plugin": "^0.1.5"
+  },
+  "dependencies": {
+    "@novnc/novnc": "^1.0.0",
```

Comment:
please don't copy all of the packages from foreman as they are not all needed here ^

Reply:
thanks, sure.. it's just that many things broke when I removed some of them.. so i wanted to make sure everything works before.